src/pages/Campaign/CampaignButtonWithOptions.tsx

```diff
@@ -60,17 +62,15 @@ export default function CampaignButtonWithOptions({
     ? campaign[type === 'swap_now' ? 'chainIds' : 'rewardChainIds'].split(',').map(Number)
     : []
 
-  const { account } = useActiveWeb3React()
+  const { account, walletSolana } = useActiveWeb3React()
   const { library } = useWeb3React()
 
   const leaderboardInfo = leaderboard || campaign?.leaderboard
 
   const refs: string[] = []
```

Comment:
dont use variable like this. it will be reset when re-render. Maybe you should use useRef here